SPECS/php-phar-io-manifest2/php-phar-io-manifest2.spec

```diff
@@ -0,0 +1,155 @@
+# remirepo/fedora spec file for php-phar-io-manifest2
+#
+# Copyright (c) 2017-2020 Remi Collet
+# License: CC-BY-SA
+# http://creativecommons.org/licenses/by-sa/4.0/
+#
+# Please, preserve the changelog entries
+#
+
+%global php_home     %{_datadir}/php
+
+Name:           php-phar-io-manifest2
+Version:        2.0.3
+Release:        4%{?dist}
+Summary:        Component for reading phar.io manifest information
+
+License:        BSD
+URL:            https://github.com/phar-io/manifest
+Source0:        https://github.com/phar-io/manifest/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#Source1:        makesrc.sh
+
+BuildArch:      noarch
+BuildRequires:  php(language) >= 7.2
+BuildRequires:  php-dom
+BuildRequires:  php-phar
+BuildRequires: (php-composer(phar-io/version) >= 3.0.1 with php-composer(phar-io/version) <  4)
+BuildRequires:  php-filter
+BuildRequires:  php-libxml
+BuildRequires:  php-pcre
+BuildRequires:  php-spl
+BuildRequires:  php-xmlwriter
+BuildRequires:  php-fedora-autoloader-devel >= 1.0.0
+%if %{with_check}
+%if 0%{?fedora} >= 32
```

Comment:
Please remove all references to fedora from the spec. Kindly re-evaluate the if-else statements wherever applicable, thanks